src/widgets/shared-utils.tsx

```diff
@@ -30,6 +30,11 @@ export function openWorkspaceTab<
   }
 }
 
+export const capitalize = s => {
```

Comment:
Keep in mind that we have `lodash-es` as a dependency, which comes with this function.

Reply:
🤙